tcf_website/templates/base/navbar.html

```diff
@@ -8,7 +8,7 @@
   <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
 
     <!-- tCF logo -->
-    <a class="navbar-brand logo-container" href="/">
+    <a class="navbar-brand logo-container mx-auto" href="/">
       <img class="logo" src="{% static 'base/img/new_logo.png' %}">
```

Comment:
Can you add alt text to the logo?
```suggestion
      <img class="logo" alt="theCourseForum home" src="{% static 'base/img/new_logo.png' %}">
```

Reply:
done